src/libretro/libretro.cpp

```diff
@@ -31,6 +31,7 @@
 #include "GPU.h"
 #include "SPU.h"
 #include "libretro.h"
+#include "streams/file_stream.h"
```

Comment:
Use system header includes here instead.

Reply:
I changed the include as requested. Any chance we can merge or is there anything else you want me to change?